packages/slo-stat-bot/src/slo-lint.ts

```diff
@@ -145,18 +147,57 @@ async function getFileShaContents(
  */
 export const lint = async function lint(
   schema: JSON,
-  sloData: JSON
+  sloData: SLORules[]
 ): Promise<ValidationResults> {
   const ajv = new Ajv();
   const validate = ajv.compile(schema);
-  const isValid = await validate(sloData);
+  let isValid = await validate(sloData);
+
+  if (validate.errors) {
+    return {
+      isValid: isValid,
+      errors: validate.errors,
+    } as ValidationResults;
+  }
+
+  //Check for contradicting SLOs
+  let errors = null;
+  for (const slo of sloData) {
+    isValid = !(await sloContradicts(slo));
+
+    if (!isValid) {
+      const sloString = JSON.stringify(slo);
+      errors = {
+        message: `Same label is in both gitHubLabels and excludedGitHubLabels for slo:  ${sloString}`,
+      };
+      break;
+    }
+  }
 
   return {
     isValid: isValid,
-    errors: validate.errors,
+    errors: errors,
   } as ValidationResults;
 };
 
+/**
+ * Function checks to see if there is a label in both gitHubLabels and excludedGitHubLabels
+ * @param slo rule
+ * @returns boolean value if slo is self contradicting
+ */
+async function sloContradicts(slo: SLORules): Promise<boolean | undefined> {
+  const githubLabels = await convertToArray(slo.appliesTo.gitHubLabels);
+  const excludedGitHubLabels = await convertToArray(
+    slo.appliesTo.excludedGitHubLabels
+  );
+
+  const isInValid = excludedGitHubLabels?.some((label: string) =>
```

Comment:
If `excludedGitHubLabels` is of type `string[]`, you shouldn't need to explicitly define the type in the arrow function.  `.some(label => githubLabels?.includes(label))` should do just fine.